### DISCUSSION TITLE
Unclear How to Get HTML Content to show in App

### DISCUSSION THREAD

**Discussion by meganjkurka** [h2oai/wave#712] [Q&A]
It is not clear how to get html content to show in an app when zones are used.  It would be nice if an example was added to the documentation to show the user the recommended behavior.  

See this issue for details: https://github.com/h2oai/wave/issues/709

**Comment by lo5** [comment]
Browsers don't resize pages based on the content of an `iframe`. Use `markup_card()` instead of `frame_card()` to have the page resize to your HTML content. See: https://h2oai.github.io/wave/docs/cards#content for notes on which card to use based on your situation.
